hasjob/static/sass/_header.sass

```diff
@@ -112,6 +112,8 @@ header.long-title
 
 .filters
   margin-bottom: 5px
+  ul.dropdown-menu
+    max-height: 68vh
```

Comment:
Apparently `vh` units are broken in iOS7, so we should expect some weirdness on an iPad. Don't have an iOS7 iPad handy to test.